src/getUsersFromDatabase.js

```diff
@@ -0,0 +1,21 @@
+exports.getUserFromDatabase = function (username) {
+
+    var AWS = require('aws-sdk');
+    AWS.config.update({ region: 'us-east-1' });
+    var docClient = new AWS.DynamoDB.DocumentClient({ apiVersion: '2012-08-10' });
```

Comment:
can you please version cloud formation file as well into this repo. later we can automate infrastructure creation